env.d/development.dist

```diff
@@ -0,0 +1,26 @@
+# Django
+DJANGO_SETTINGS_MODULE=marsha.settings
+DJANGO_CONFIGURATION=Development
+DJANGO_SECRET_KEY=ThisIsAnExampleKeyForDevPurposeOnly
+DJANGO_JWT_SIGNING_KEY=ThisIsAnExampleKeyForDevPurposeOnly
+DJANGO_BYPASS_LTI_VERIFICATION=True
+
+# Database
+POSTGRES_HOST=db
+POSTGRES_DB=marsha
+POSTGRES_USER=marsha_user
+POSTGRES_PASSWORD=pass
+
+# Python
+PYTHONUNBUFFERED=1
+
+#  AWS Admin role for development purposes
+DJANGO_AWS_ACCESS_KEY_ID=yourAwsAccesKey
+DJANGO_AWS_SECRET_ACCESS_KEY=YourAwsSecretAccessKey
+DJANGO_AWS_DEFAULT_REGION=eu-west-1
+DJANGO_AWS_SOURCE_BUCKET_NAME=dev-marsha-source
+
+DJANGO_CLOUDFRONT_URL=
```

Comment:
This one should be removed I think and replace by `CLOUDFRONT_SIGNED_URLS_ACTIVE` set to false.

Reply:
in fact we should made what is described here https://github.com/openfun/marsha/pull/124/files#r231504945

`